### PR TITLE
what good are canaries without a coalmine

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

**Comment by jwmatthews** [comment]
This is some impressive wizardry!

Would you add variables for the controller-manager and apiserver images?
Thought is if we have parameters it will help us to test local controller-manager builds for testing.

I like how enabling coalmine triggers this flow.

**Comment by eriknelson** [comment]
I'm giving this a try with catalog images I'm building from source.

File: config/my_vars.yml.example [hunk 1]
```diff
@@ -14,4 +14,9 @@ dockerhub_org_name: example_org
...
 
 # Default to the "anyuid" scc for pod deployments
-# scc_anyuid: True
\ No newline at end of file
+# scc_anyuid: True
+
+# Use the quay.io canary images for bleeding edge development
+# coalmine: true
+# coalmine_apiserver: quay.io/kubernetes-service-catalog/apiserver:canary
+# colamine_controller: quay.io/kubernetes-service-catalog/controller-manager:canary
```
Comment (eriknelson) [inline config/my_vars.yml.example]: Typo on this `colamine` -> `coalmine`.

**Comment by eriknelson** [comment]
Successfully deployed my catalog from source and confirmed I'm able to provision. Awesome job Jason :+1:

**Comment by jmontleon** [comment]
The updated version has options for the images.

**Comment by fabianvf** [comment]
+1 just for title :shipit:

**Pull request merged**